"""
Connect to an existing data base
This case is for MySQL
"""


from getpass import getpass
from mysql.connector import connect, Error

try:
    with connect(
        host="localhost",
        user=input("Enter username: "),
        password=getpass("Enter password: "),
        database="name_of_db",
    ) as connection:
        print(connection)
except Error as e:
    print(e)
